### PR TITLE
Add one-time login option.

### PR DESCRIPTION
**Pull request by ordavidil**

### CODE REVIEW

File: src/Restful/Login.elm [hunk 1]
```diff
@@ -1,45 +1,18 @@
...
-        , mapAuthenticatedData
-        , mapBoth
-        , maybeAnonymousData
-        , maybeAuthenticatedData
-        , recordLogin
-        , tryAccessToken
-        , tryLogin
-        , update
-        )
+module Restful.Login exposing
```
Comment (amitaibu) [inline src/Restful/Login.elm]: which elm-format version are you using here?

Comment (ordavidil) [reply src/Restful/Login.elm]: elm-format 0.8.1

used with `--elm-version=0.18`

Comment (ordavidil) [reply src/Restful/Login.elm]: Must say, when I've tried with `--elm-version=0.19` the result was the same...

**Comment by amitaibu** [comment]
@ordavidil can you please merge and try to update the published package - https://medium.com/@Max_Goldstein/how-to-publish-an-elm-package-3053b771e545

**Comment by ordavidil** [comment]
> @ordavidil can you please merge and try to update the published package - https://medium.com/@Max_Goldstein/how-to-publish-an-elm-package-3053b771e545

@amitaibu 
that's for `elm 0.19` but this package is `0.18`, should I follow that anyway?

**Comment by amitaibu** [comment]
I don't know - what I care is that it needs to be published :)

**Pull request merged**